app/controllers/responses_controller.rb

```diff
@@ -0,0 +1,27 @@
+class ResponsesController < ApplicationController
+  
+  def new
+    @project_request = ProjectRequest.find_by_id(params[:project_request_id])
```

Comment:
Is this fine in the cases where find_by_id fails? Specifically, will the nil @project_request variable be handled appropriately?